network/network_windows_test.go

```diff
@@ -227,3 +229,46 @@ func TestDeleteNetworkImplHnsV1WithTimeout(t *testing.T) {
 		t.Fatal("Failed to timeout HNS calls for deleting network")
 	}
 }
+
+func TestAddIPv6DefaultRoute(t *testing.T) {
+	_, ipnetv4, _ := net.ParseCIDR("10.240.0.0/12")
+	_, ipnetv6, _ := net.ParseCIDR("fc00::/64")
+
+	nm := &networkManager{
+		ExternalInterfaces: map[string]*externalInterface{},
+		plClient:           platform.NewMockExecClient(false),
+	}
+
+	networkSubnetInfo := []SubnetInfo{
+		{
+			Family:  platform.AfINET,
+			Gateway: net.ParseIP("10.240.0.1"),
+			Prefix:  *ipnetv4,
+		},
+		{
+			Family:  platform.AfINET6,
+			Gateway: net.ParseIP("fc00::1"),
+			Prefix:  *ipnetv6,
+		},
+	}
+
+	nwInfo := &NetworkInfo{
+		Id:           "d3f97a83-ba4c-45d5-ba88-dc56757ece28",
+		MasterIfName: "eth0",
+		Mode:         "bridge",
+		Subnets:      networkSubnetInfo,
+	}
+
+	extInterface := &externalInterface{
+		Name:    "eth0",
+		Subnets: []string{"subnet1", "subnet2"},
+	}
+
+	Hnsv2 = hnswrapper.NewHnsv2wrapperFake()
+
+	// check if network can be successfully created
+	_, err := nm.newNetworkImplHnsV2(nwInfo, extInterface)
```

Comment:
this UT doesn't test whole code path.. it should test all code paths.. you may need to return failure and then only it will go to path where it attempts to add route